acbuild/acbuild.go

```diff
@@ -239,6 +244,12 @@ func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd
 }
 
 func main() {
+	if path.Base(os.Args[0]) == "acbuild-script" {
+		os.Args[0] = path.Join(path.Dir(os.Args[0]), "acbuild")
+		args := os.Args[1:]
+		os.Args = []string{os.Args[0], "script"}
+		os.Args = append(os.Args, args...)
+	}
```

Comment:
I think you are kind of fighting with multicall here. I was in need of something similar so I just split https://github.com/coreos/rkt/pull/2815 of my working branch. Do you think it could make things easier here?